dashboard-object-viewer/package.json

```diff
@@ -14,78 +14,29 @@
     "dist"
   ],
   "scripts": {
-    "build": "npm run clean && webpack",
-    "clean": "rimraf dist/*",
-    "lint": "eslint './src/**/*.{js,jsx,ts,tsx}' --quiet",
-    "start": "webpack watch",
-    "test": "jest --coverage --watchAll --config=configs/jest.json"
+    "start": "vite build --watch",
+    "build": "vite build"
   },
   "dependencies": {
-    "react-json-view": "^1.21.3"
+    "@deephaven/components": "0.34.0",
+    "@deephaven/dashboard": "0.34.0",
+    "@deephaven/jsapi-shim": "0.34.0",
+    "@deephaven/log": "0.34.0",
```

Comment:
They aren't being marked external so they aren't really peer deps. If we allowed you to mark our packages as external, then they could be peers if we wanted to just use whatever the installed version of our packages was on the server. Double edged sword of automatically up to date, but also automatically breaking the plugin

Also, for distributed bundles the difference between regular/peer/dev deps is effectively nothing. Aside from eslint rules which say we need to list in regular deps. But _technically_ everything is actually a dev dep since we're bundling them up. If a user did `npm install @deephaven/js-plugin-object-viewer` or whatever, the only thing the need is the bundled file. Installing the other dependencies won't affect them at all

In our Docker images in core, we pull the tgz directly from npm which bypasses downloading dependencies (since we don't need them in the web environment)